javascript/express/security/audit/remote-property-injection.yaml

```diff
@@ -27,18 +27,30 @@ rules:
               - pattern-inside: function ... ($REQ, $RES) {...}
               - pattern-inside: function ... ($REQ, $RES, $NEXT) {...}
               - patterns:
-                  - pattern-either:
-                      - pattern-inside: $APP.$METHOD(..., function $FUNC($REQ, $RES) {...})
-                      - pattern-inside: $APP.$METHOD(..., function $FUNC($REQ, $RES, $NEXT) {...})
```

Comment:
You have removed the case with`$NEXT` here, but have left it in in the other rules. Is that on purpose?

Reply:
i think the pattern is the same i just somehow messed up the ordering